webpack.config.js

```diff
@@ -0,0 +1,68 @@
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+
+module.exports = {
+    entry: {
+        'lemon-reset': './src/index.js',
+    },
+    output: {
+        library: 'LemonReset',
+        libraryTarget: 'umd',
+    },
+    externals: {
+        react: {
+            commonjs: 'react',
+            commonjs2: 'react',
+            amd: 'React',
+            root: 'React',
+        },
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        babelrc: true,
+                    },
+                },
+            },
+            {
+                test: /\.css$/,
+                use: [
+                    MiniCssExtractPlugin.loader,
+                    {
+                        loader: 'css-loader',
+                        options: {
+                            modules: true,
+                            localIdentName: '[local]--[hash:base64:5]',
+                        },
+                    },
+                ],
+            },
+        ],
+    },
+    optimization: {
+        minimizer: [
+            new UglifyJsPlugin({
+                cache: true,
+                parallel: true,
+                sourceMap: true,
+                uglifyOptions: {
+                    output: {
+                        comments: false,
+                    },
+                },
+            }),
+            new OptimizeCSSAssetsPlugin({}),
+        ],
+    },
+    plugins: [
+        new MiniCssExtractPlugin({
+            filename: '[name].css',
```

Comment:
I'm thinking not. [CDNjs contributing guidelines](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#d-adding-a-new-library-with-its-assets) say not to do this, and it will make it trickier to set up Webpack to use the UMD distribution. I was thinking that a Webpack build could be configured to use this dist with a config like this:
```
    resolve: {
      alias: {
        'lemon-reset$': 'lemon-reset/dist/lemon-reset.js',
      }
    },
```

If the version was in the filename, we'd have to update this config whenever lemon-reset is bumped.